docs/README.md

```diff
@@ -70,41 +77,85 @@ root = "./app"
+The Python environment may be explicitly specified using the
+[`environment.python`](./reference/configuration.md#python) setting or
+[`--python`](./reference/cli.md#ty-check--python) flag.
+
+When setting the environment explicitly, non-virtual environments can be provided.
 
 ## Python version
 
-The supported Python syntax and standard library functions differ between Python versions. For example, Python 3.10 introduced support for `match` statements and the `sys.stdlib_module_names` symbol. While these features enhance the language, using them in a project targeting an older Python version can lead to compatibility issues.
-ty helps you avoid such compatibility issues by checking your code against the Python version your project targets, and flagging any use of features or standard library symbols not available in that version.
+The Python version affects allowed syntax, type definitions of the standard library, and type
+definitions of first- and third-party modules that are conditional on the Python version.
+
+For example, Python 3.10 introduced support for `match` statements and added the the
+`sys.stdlib_module_names` symbol. However, if your project also supports Python 3.9, you cannot use
+these new features.
+
+By default, the lower bound of the project's [`requires-python`](<(https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#python-requires)>) field (from the `pyproject.toml`) is
+used as the target Python version, ensuring that features and symbols only available in newer Python
+versions are not used.
 
-ty determines the project's target Python version from:
+If the `requires-python` field is not available, the latest stable version supported by ty is used,
+which is currently 3.13.
 
-- The [`python-version`](./reference/configuration.md#python-version) value in your configuration or the [`--python-version`](./reference/cli.md#ty-check--python-version) command line option.
-- The lower bound of the project's [`requires-python`](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#python-requires) field in your `pyproject.toml`.
-- If neither is specified, ty defaults to the latest stable Python version it supports (Python 3.13)
+ty will not infer the Python version from the Python environment at this time.
+
+The Python version may be explicitly specified using the
+[`python-version`](./reference/configuration.md#python-version) setting or the
+[`--python-version`](./reference/cli.md#ty-check--python-version) flag.
```

Comment:
it's a bit weird to me that we mention these last, since they're the things that ty checks for first